packages/ra-ui-materialui/src/input/FileInput.tsx

```diff
@@ -48,6 +49,8 @@ export const FileInput = (props: FileInputProps) => {
         source,
         validate,
         validateFileRemoval,
+        disabled,
+        readOnly,
```

Comment:
**thought**: if we follow the [HTML spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly), `readOnly` makes no sens in `file` input type. As stated before, we should remove it. @slax57 ?

Reply:
We could still need it as a workaround for the RHF limitation, so even if I agree with your point, I recommend we keep it. We have it for free and it might save us later.